rules/windows/process_creation/win_malware_notpetya.yml

```diff
@@ -30,7 +30,7 @@ detection:
     rundll32_dash1:
         Image|endswith: '\rundll32.exe'
         CommandLine|endswith: '.dat,#1'
-    perfc_keyword|contains:
+    perfc_keyword:
         - '\perfc.dat'
```

Comment:
Just read the ref. I can't say I fully understand it, but it seems to me like the rundll32_dash1 condition already covers the execution of the malware, and perfc_keyword condition is just meant to look for `perfc.dat` in the entire log entry (which should be a strong enough signal on its own). 

Again, not 100% sure, but I want to make sure we are on the same page :)

Reply:
Since it uses 'process_creation', the keyword search should not be bad.
Just wait if there are FP 
As the fix is good I validate.
Thanks for the works